### DISCUSSION TITLE
Change TimeSeries Tooltip Default to `All` and not 'single`

### DISCUSSION THREAD

**Discussion by dessen-xu** [grafana/grafana#43543] [Dashboards, Panels & Visualizations]
**Steps to reproduce**:

Tooltip of TimeSeries panel displays single in default, even if migrated from old graph.

TimeSeries
![image](https://user-images.githubusercontent.com/41524013/147312695-61471c00-8497-4c0b-8e87-32b00f33be84.png)

Old Graph
![image](https://user-images.githubusercontent.com/41524013/147312320-9781fdae-4a75-465f-8f5f-dcaf439bffce.png)

Is it elaborate, or it is not very good

**Actual Result**:

**Expected Result**

Display all in default like before.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 8.2.7
- Data source type & version:
- User OS & Browser: macos, chrome
- Others:

**Comment by zuchka** [comment]
Thanks for creating this issue, @dessen-xu. We always appreciate your contributions and feedback

> even if migrated from old graph

How precisely did you migrate?

> Display all in default like before.

I'm going to mark this as a feature-request for now, since the current default behavior appears to work as intended. If you think this is a migration bug, please open a new issue and provide more information and steps to reproduce 👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
